elasticsearch-transport/lib/elasticsearch/transport/transport/http/manticore.rb

```diff
@@ -63,7 +63,7 @@ def __build_connections
               @request_options[:headers] = options[:headers]
             end
 
-            client_options = setup_ssl(options[:ssl] || {})
+            client_options = {:ssl => options[:ssl] || {}}
```

Comment:
It would be great if we could have another `@example` section with a code example of setting up SSL -- can you please do it?